File: Maths For IT/Pyramid Assignment/pyramid.py
def create_pyramid(height):
    for i in range(1, height + 1):
        for j in range(height - i):
            if j < height:
                print(" ", end="")
        
        for j in range(i):
            print("#", end="")
        
        print("")

x = input("Enter Height: ")

print(int(x))

while (int(x) < 1 or int(x) > 8 or isinstance(int(x), int) == False):
    x = input("Height must be between 1 - 8: ")

create_pyramid(int(x))

File: Maths For IT/Pyramid Assignment/pyramid.lua
function create_pyramid(height) 
    for i = 1, height do
        for j = 1, height - i do
            io.write(" ");
        end

        for j = 1, i do
            io.write("#");
        end

        io.write("\n");
    end
end

local height;

while true do
    io.write("Enter height: ");
    height = tonumber(io.read());

    if not height then
        print("Must be a integer: ");
    elseif height > 8 or height < 0 then
        print("height must be between 1 - 8: ")
    else 
        break
    end
end

create_pyramid(height)